src/demo-app/a11y/button-toggle/button-toggle-a11y.html

```diff
@@ -0,0 +1,62 @@
+<section>
+  <h2>Single button toggle</h2>
+  <md-button-toggle>Yes</md-button-toggle>
+</section>
+
+<section>
+  <h2>Button toggles with icons</h2>
+  <md-button-toggle-group name="alignment">
```

Comment:
Added `role="group"` to button-toggle-group with multi-select. Without it the aria-label does not work.